WireAPI/Tests/WireAPITests/Mocks/URLRequest+MockResponse.swift

```diff
@@ -44,4 +44,27 @@ extension URLRequest {
         return (try jsonPayload.data(), response)
     }
 
+    func mockErrorResponse(
+        statusCode: HTTPStatusCode,
+        label: String = ""
+    ) throws -> (Data, HTTPURLResponse) {
+        guard let url else {
+            throw "Unable to create mock response, request is missing url"
```

Comment:
throwing String?

Reply:
Yes, in this test target String conforms to Error so we can just throw error messages in tests.